
const UserException = {

    userRepeated: 'userRepeated',
    userNotFound: 'userNotFound'
}

module.exports = UserException;